lib/cluster.js

```diff
@@ -341,9 +341,9 @@ function masterInit() {
       process: workerProcess
     });
 
-    worker.on('message', (message, handle) =>
-      cluster.emit('message', message, handle)
-    );
+    worker.on('message', function(message, handle) {
+      cluster.emit('message', this, message, handle);
```

Comment:
Probably I am missing something. When the `message` event is emitted by `cluster`, the `this` will be `cluster`, not the worker object, right?